.github/workflows/tests_linux.yml

```diff
@@ -44,191 +43,107 @@ jobs:
           pip uninstall pennylane -y
           pip install git+https://github.com/PennyLaneAI/pennylane.git
-      - name: Install ML libraries for interfaces
-        run: |
-          pip install --upgrade torch==${{ env.TORCH_VERSION }} -f https://download.pytorch.org/whl/cpu/torch_stable.html
-          pip install --upgrade "jax[cpu]"  # This also installs jaxlib
-          pip install --upgrade tensorflow~=${{ env.TF_VERSION }}
-      - name: Install lightning.kokkos device
-        run: |
-          cd main
-          CXX=$(which g++-10) BACKEND="OPENMP" python -m pip install -e . --verbose
-      - name: Run PennyLane-Lightning unit tests
-        run: |
-          cd main/
-          pytest tests/
-          pl-device-test --device lightning.kokkos
-
-  cpptests_threads:
-    name: C++ tests (Linux) for Threads Kokkos backend
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-20.04]
-    steps:
-      - name: Cancel previous runs
-        uses: styfle/cancel-workflow-action@0.4.1
-        with:
-          access_token: ${{ github.token }}
-
-      - uses: actions/setup-python@v2
-        name: Install Python
-        with:
-          python-version: '3.7'
-
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 2 # for codecov
-
-      - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get -y -q install cmake gcc-10 g++-10 gcovr lcov
-
-      - name: Build and run unit tests
-        run: |
-            cmake . -BBuild -DKokkos_ENABLE_THREADS=ON -DPLKOKKOS_BUILD_TESTS=ON -DPLKOKKOS_ENABLE_SANITIZER=ON -DCMAKE_CXX_COMPILER="$(which g++-${{ env.GCC_VERSION }})"
-            cmake --build ./Build
-            ./Build/pennylane_lightning_kokkos/src/tests/runner_kokkos
-
-  pythontests_threads:
-    name: Python tests for Threads Kokkos Backend
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-20.04]
-
-    steps:
-      - name: Checkout PennyLane-Lightning-Kokkos
-        uses: actions/checkout@v2
-        with:
-          path: main
-          fetch-depth: 2
-
-      - uses: actions/setup-python@v2
-        name: Install Python
-        with:
-          python-version: '3.7'
-
-      - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get -y -q install cmake gcc-10 g++-10
 
-      - name: Get required Python packages
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install pytest pytest-mock flaky
-          pip uninstall pennylane -y
-          pip install git+https://github.com/PennyLaneAI/pennylane.git
-      - name: Install ML libraries for interfaces
-        run: |
-          pip install --upgrade torch==${{ env.TORCH_VERSION }} -f https://download.pytorch.org/whl/cpu/torch_stable.html
-          pip install --upgrade "jax[cpu]"  # This also installs jaxlib
-          pip install --upgrade tensorflow~=${{ env.TF_VERSION }}
```

Comment:
Do we still need these for the Python suite tests?

Reply:
Following the L-GPU's implementation, I think we don't need these for the python tests.